trackpy/feature.py

```diff
@@ -471,7 +486,7 @@ def locate(raw_image, diameter, minmass=100., maxsize=None, separation=None,
     DataFrame([x, y, mass, size, ecc, signal])
         where mass means total integrated brightness of the blob,
         size means the radius of gyration of its Gaussian-like profile,
-        and ecc is its eccentricity (1 is circular).
+        and ecc is its eccentricity (0 is circular).
```

Comment:
Yes, you always get +-0 for round features. It was correct on the `refine` docstring.